explainaboard/utils/preprocessor.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+import re
+import string
+import sys
+import unicodedata
+
+from explainaboard.utils.tokenizer import MLQAMixTokenizer, SingleSpaceTokenizer
+
+
+class Preprocessor:
+    def __call__(self, text: str) -> str:
```

Comment:
Add `@abc.abstractmethod` decorator, and replace `raise NotImplementedError` with `...`

Reply:
sounds good, do we want to do a similar thing for the `tokenizer` class?

Reply:
Yeah, good idea. I just recently learned this syntax from @odashi :)